recipes/stir/meta.yaml

```diff
@@ -0,0 +1,66 @@
+
+{% set name = "STIR" %}
+{% set version = "3.0.1.dev1" %}
+{% set sha256 = "8bd7d884917c23340f5cac7830dfeb75b6a3bb3d7279b296c92bb38a87281f89" %}
+{% set treehash = "add396c3c6c575a5e23f42fabcadbe5c11e69d0a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ treehash }}.zip
+  url: https://github.com/UCL/{{ name }}/archive/{{ treehash }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py2k]
+  skip: true  # [win and py<36]
+  number: 1
+
+requirements:
+  build:
```

Comment:
Can you change this to the [new compiler syntax](https://conda-forge.org/docs/meta.html#compilers)?

I think this is what you need:
```yaml
requirements:
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - cmake
  host:
    - python
    - boost-cpp
    - numpy
    - swig
  run:
    - python
    - {{ pin_compatible('numpy') }}
```